src/styles/blocks/card.scss

```diff
@@ -0,0 +1,80 @@
+@import '/src/styles/blocks/variables';
+
+.card {
+  width: 200px;
+  box-sizing: border-box;
+  background-color: rgba(255, 255, 255, 1);
+  border: 1px solid rgba(243, 243, 243, 1);
+  font-family: $main-font-family;
+  display: flex;
+  flex-direction: column;
+  padding: 32px 16px 16px;
+  border-radius: 5px;
+
+  &__photo {
+    background-image: url(../images/imac.jpeg);
+    background-position: center;
+    background-size: contain;
+    background-size: 100% 100%;
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+  }
+
+  &__name {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: rgba(6, 11, 53, 1);
+    text-align: left;
+    margin: 0 0 4px;
+  }
+
+  &__code {
+    font-weight: 400px;
```

Comment:
The value of the `font-weight` property should not have a unit. It should be `font-weight: 400;` instead of `font-weight: 400px;`.